tools/test-js-with-node

```diff
@@ -241,6 +242,7 @@ EXEMPT_FILES = make_set(
         "web/src/stream_settings_containers.ts",
         "web/src/stream_settings_ui.js",
         "web/src/stream_ui_updates.js",
+        "web/src/sub_store.ts",
```

Comment:
So I mentioned this for `stream_data.ts`, but this should also be address instead by changing the tests.

More generally, this commit shouldn't exist -- each individual commit should pass all tests, and the changes to tests required

Check out the [Zulip commit message](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details -- but one reason this practice is important is that it makes it possible to merge or revert individual commits.

Reply:
Okay, I will remember it for future pull requests.